repo/packages/O/openldap-admin/2/marathon.json.mustache

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{openldap-admin.cpus}},
+    "mem": {{openldap-admin.mem}},
+    "instances": 1,
+    "env": {
+        "LDAP_SERVER_HOST": "{{networking.ldap_host}}.marathon.l4lb.thisdcos.directory",
+        "LDAP_SERVER_PORT": "{{networking.port_ldap_server}}"
+    },
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.openldap-admin-docker}}",
+            "forcePullImage": false,
+            "network": "BRIDGE",
+            "portMappings": [
+            {
+                "containerPort": 80,
+                "hostPort": 0,
+                {{#networking.external_access}}
+                "servicePort": {{networking.port_external}},
```

Comment:
I'm defaulting to external_access=false so that the VIP won't be created. I can see use cases where:
- AdminRouter is used exclusively for Dan access.
- AdminRouter is bloated with traffic
- The cluster is used for "outside" access for people not exposed at all to DC/OS

... so I thought it'd be a good idea to give the *option* to automatically put it in MLB